runners/spark/pom.xml

```diff
@@ -200,9 +201,8 @@
       <scope>provided</scope>
     </dependency>
     <dependency>
-      <groupId>commons-lang</groupId>
-      <artifactId>commons-lang</artifactId>
-      <version>2.6</version>
```

Comment:
What does "align" mean here -- align with what? Is the claim that Spark already depends on `lang3` so we should just use it, e.g.?

How does `mvn dependency:list` change with this switch?